google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -42,6 +42,14 @@
 _sql_api_version: str = "v1beta4"
 
 
+class ConnectionSSLContext(ssl.SSLContext):
+    """Subclass of ssl.SS:Context with added request_ssl attribute."""
```

Comment:
Is there a typo here?  I think a little more explanation on why this class is used in the code would be good as well.

Reply:
Good catch on the typo. And yeah, I can expand on this a bit more. Does the class name look ok? I wasn't sure what to call it.